WowPacketParser/Enums/UpdateFields.cs

```diff
@@ -142,30 +148,46 @@ public enum UnitField
         UNIT_FIELD_BATTLE_PET_COMPANION_GUID,
         UNIT_FIELD_BATTLE_PET_DB_ID,
         UNIT_FIELD_BONUS_RESISTANCE_MODS,
+        [UpdateField(UpdateFieldType.Float)]
         UNIT_FIELD_BOUNDINGRADIUS,
+        [UpdateField(UpdateFieldType.Bytes)]
         UNIT_FIELD_BYTES_0,
+        [UpdateField(UpdateFieldType.Bytes)]
         UNIT_FIELD_BYTES_1,
+        [UpdateField(UpdateFieldType.Bytes)]
         UNIT_FIELD_BYTES_2,
         UNIT_FIELD_CHANNEL_DATA,
+        [UpdateField(UpdateFieldType.Guid)]
         UNIT_FIELD_CHANNEL_OBJECT,
         UNIT_FIELD_CHARM,
+        [UpdateField(UpdateFieldType.Guid)]
         UNIT_FIELD_CHARMEDBY,
         UNIT_FIELD_CONTENT_TUNING_ID,
+        [UpdateField(UpdateFieldType.Float)]
         UNIT_FIELD_COMBATREACH,
+        [UpdateField(UpdateFieldType.Guid)]
         UNIT_FIELD_CREATEDBY,
         UNIT_FIELD_CRITTER,
+        [UpdateField(UpdateFieldType.Guid)]
         UNIT_FIELD_DEMON_CREATOR,
         UNIT_FIELD_DISPLAYID,
         UNIT_FIELD_DISPLAY_POWER,
         UNIT_FIELD_DISPLAY_SCALE,
         UNIT_FIELD_EFFECTIVE_LEVEL,
         UNIT_FIELD_END,
+        [UpdateField(UpdateFieldType.Uint)]
+        [UpdateField(UpdateFieldType.Custom, ClientVersionBuild.V6_2_3_20726)]
```

Comment:
not sure why you would flag this as custom only in 6.2.3+, its always been a reference to FactionTemplate.dbc/db2

Reply:
Because this hack was only in WOD+ update handler. So I preserved old behaviour.